libvips/iofuncs/util.c

```diff
@@ -1593,10 +1593,20 @@ vips_amiMSBfirst(void)
 #endif
 }
 
+static void
+vips__swap_chars(char *buf, char from, char to)
+{
+    int i;
+
+    for (i = 0; buf[i]; i++)
+        if (buf[i] == from)
+            buf[i] = to;
+}
+
```

Comment:
This can be removed if we use [`g_strdelimit()`](https://docs.gtk.org/glib/func.strdelimit.html).

```suggestion
```